fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+            :attr:`required_keys` to field names of their dataset that contain
+            the required values
+        transform (None): SAM specific image transform function to apply
+        use_numpy (False): whether to use numpy arrays rather than PIL images
+            and Torch tensors when loading data
+        box_transform (None): SAM specific box transform function to apply
+        point_transform (None): SAM specific point transform function to apply
+        text_prompts (None): Text prompts for concept prompting the model
+        operation_mode ("concept"): Operation mode of the model (required for collate_fn)
+    """
+
+    def __init__(
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)